### ISSUE TITLE
Spec-parser errors

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-3-model#443]
Since merging in PR #441 we're getting the following error from the spec-parser:

```
Traceback (most recent call last):
  File "/home/runner/work/spdx-3-model/spdx-3-model/spec-parser/main.py", line 71, in <module>
    spec.gen_rdf()
  File "/home/runner/work/spdx-3-model/spdx-3-model/spec-parser/spec_parser/utils.py", line 188, in gen_rdf
    convert_spdx_owl_to_jsonld_context(jsonld_file_name, self.args["out_dir"])
  File "/home/runner/work/spdx-3-model/spdx-3-model/spec-parser/spec_parser/owl_to_context.py", line 50, in convert_spdx_owl_to_jsonld_context
    type_id = node["rdfs:range"]["@id"]
              ~~~~~~~~~~~~~~~~~~^^^^^^^
TypeError: string indices must be integers, not 'str'
Error: Process completed with exit code 1.
```

**Comment by goneall** [comment]
@armintaenzertng - Can you take a look?  I don't think it is related to the PR since the file changed shouldn't even be parsed.

**Comment by zvr** [comment]
I think it was because of the merge of https://github.com/spdx/spec-parser/pull/58.

I might take a look tomorrow, as I'm working on the spec-parser anyway.

FYI, @armintaenzertng will be back after 15th.

**Comment by goneall** [comment]
> I think it was because of the merge of [spdx/spec-parser#58](https://github.com/spdx/spec-parser/pull/58).
> 
> I might take a look tomorrow, as I'm working on the spec-parser anyway.
> 
> FYI, @armintaenzertng will be back after 15th.

Thanks @zvr

**Comment by zvr** [comment]
OK, little debugging before midnight:
it fails because the `/Software/declaredLicense` property has `Range: /Licensing/AnyLicenseInfo` and there is no more `/Licensing` namespace...

@goneall you probably know what to do.

I should point out that these will eventually be Relationships, so they might as well get deleted and two RelationshipTypes be added.

**Comment by goneall** [comment]
> OK, little debugging before midnight

Thanks @zvr  - I'll take care of it - I'll create the PR to fix the failures and RelationshipTypes.

**Comment by goneall** [comment]
Resolved with PR #448
